plugins/available/nvm.plugin.bash

```diff
@@ -6,7 +6,13 @@ cite about-plugin
 about-plugin 'node version manager configuration'
 
 export NVM_DIR="$HOME/.nvm"
```

Comment:
Ah, got it, thanks! So we run `nvm.sh` from the brew prefix, but still leave `NVM_DIR` set to the home folder so that any packages the user decides to install end up there.